app/controllers/auth/sessions_controller.rb

```diff
@@ -0,0 +1,14 @@
+class Auth::SessionsController < ApplicationController
+
+  def create
+    strava_request = "https://www.strava.com/oauth/token?client_id=#{ENV["strava_client_id"]}&client_secret=#{ENV["strava_client_secret"]}&code=#{params["code"]}"
+    strava_response = Faraday.post(strava_request)
+    user_data = JSON.parse(strava_response.body)
```

Comment:
Yes!  I agree - that's the step I'm working on.  This was the ugly bit just to see how it works.  Good to know I'm headed in the right direction.